stylesheets/_touch-friendly-links.scss

```diff
@@ -0,0 +1,10 @@
+// This mixin will expand the clickable area for a link on mobile and tablet to make it easier to click on a touch
```

Comment:
Possibly a bit pedantic, but I can't see anything that limits this behaviour to only 'on mobile and tablet'? May be worth rewording to remove that clause.